cocos/core/assets/prefab.ts

```diff
@@ -61,9 +63,11 @@ const OptimizationPolicy = Enum({
      * @en Optimize for creating instances multiple times.<br>
      * This option enables code generation for this prefab.
      * When this prefab will usually create multiple instances, please select this option.
-     * It is also recommended to select this option if the prefab instance in the scene has Auto Sync enabled and there are multiple instances in the scene.
+     * It is also recommended to select this option if the prefab instance in the scene has
+     * Auto Sync enabled and there are multiple instances in the scene.
      * @zh 优化多次创建性能。<br>
-     * 该选项会启用针对这个 prefab 的代码生成优化操作。当该 prefab 加载后，一般会创建多个实例时，请选择此项。如果该 prefab 在场景中的节点启用了自动关联，并且在场景中有多份实例，也建议选择此项。
+     * 该选项会启用针对这个 prefab 的代码生成优化操作。当该 prefab 加载后，一般会创建多个实例时，请选择此项。
+     * 如果该 prefab 在场景中的节点启用了自动关联，并且在场景中有多份实例，也建议选择此项。
```

Comment:
Dont add line break in Chinese api, or it will add a redundant space in the document.

Reply:
the CI show warnings...